docs/manual/migration-guide.md

```diff
@@ -0,0 +1,79 @@
+# Migration Guide
+
+This guide details the main steps to migrate between major release versions.
+
+## Migrate from 1.x to 2.x
+
+The 2.0 release introduces some significant changes in the API compared to the 1.0 release in order to include support for multiple media tracks per peer connection.
+
+### C# library
+
+The C# library exposes a transceiver API very similar to the one found in the WebRTC 1.0 standard, and therefore familiarity with that standard helps understanding the API model. The API is not guaranteed to exactly follow the standard, but generally stays pretty close to it.
+
+#### Standalone track objects
+
+Audio and video tracks are now standalone objects, not tied to a peer connection.
+
+- Users must create the audio and video track objects explicitly, independently of the peer connection.
+  - Audio tracks are created with class methods such as [`LocalAudioTrack.CreateFromDeviceAsync()`](xref:Microsoft.MixedReality.WebRTC.LocalAudioTrack.CreateFromDeviceAsync(Microsoft.MixedReality.WebRTC.LocalAudioTrackSettings)).
+  - Video tracks are created with class methods such as [`LocalVideoTrack.CreateFromDeviceAsync()`](xref:Microsoft.MixedReality.WebRTC.LocalVideoTrack.CreateFromDeviceAsync(Microsoft.MixedReality.WebRTC.LocalVideoTrackSettings)).
+- Users are owning those track objects and must ensure they stay alive while in use by the peer connection, and are properly disposed after use (`IDisposable`).
+
+#### Transceivers
+
+Previously in 1.0 the peer connection was based on an API similar to the track-based API of the pre-standard WebRTC specification. The 2.0 release introduces a different _tranceiver_-based API for manipulating audio and video track, which is more closely based on the WebRTC 1.0 standard.
+
+- A _transceiver_ is a "media pipe" in charge of the encoding and transport of some audio or video tracks (at most one each way, sent and received).
```

Comment:
> some audio or video tracks (at most one each way, sent and received)
Maybe worth being more explicit about there being at most 1 of each kind here? Something like "a transceiver has 4 slots - incoming audio/incoming video/outgoing video/outgoing audio. Each slot may be unassigned or assigned to a track"

Reply:
Or maybe not, because that's wrong 😉 A transceiver has some media kind (audio or video), a sender track, and a receiver track. I will try to reword it.

Reply:
Fixed.